src/components/shared/button/Button.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import * as Styled from "./Button.styled";
+
+interface ButtonProps {
+  isFormVisible: boolean;
+  label?: string;
+  onClick: Function;
+  type?: "Primary" | "Secondary" | "Tertiary";
```

Comment:
Just a suggestion, I prefer naming button types after their action. Often the style is decided by actions like `remove` or `add`

Reply:
Doenja said the exact opposite :p

Reply:
🍿

Reply:
I agree with Doenja though.

Reply:
I've checked around for convention and `primary` and `secondary` are definitely most common. It comes from the idea that buttons types have a relation with each other. The primary action is utilized more as the user's expected task